public/app/features/provisioning/Wizard/SynchronizeStep.tsx

```diff
@@ -7,32 +7,36 @@ import { Job, useCreateRepositoryJobsMutation } from 'app/api/clients/provisioni
 
 import { JobStatus } from '../Job/JobStatus';
 
-import { StepStatusInfo, WizardFormData } from './types';
+import { useStepStatus } from './StepStatusContext';
+import { WizardFormData } from './types';
 
 export interface SynchronizeStepProps {
-  onStepStatusUpdate: (info: StepStatusInfo) => void;
   requiresMigration: boolean;
 }
 
-export function SynchronizeStep({ onStepStatusUpdate, requiresMigration }: SynchronizeStepProps) {
+export function SynchronizeStep({ requiresMigration }: SynchronizeStepProps) {
+  const [job, setJob] = useState<Job>();
+
+  const { setStepStatusInfo } = useStepStatus();
   const [createJob] = useCreateRepositoryJobsMutation();
   const { getValues, register, watch } = useFormContext<WizardFormData>();
   const repoType = watch('repository.type');
-  const supportsHistory = requiresMigration && repoType === 'github';
-  const [job, setJob] = useState<Job>();
   const { t } = useTranslate();
+
+  const supportsHistory = requiresMigration && repoType === 'github';
```

Comment:
You'll need to sync with the latest main to incorporate the changes from https://github.com/grafana/grafana/pull/106005.